kentikapi/client_integration_test.go

```diff
@@ -149,9 +148,9 @@ func TestClient_GetUserWithRetries(t *testing.T) {
 			}
 
```

Comment:
`default timeout is longer than 30 ms` and `timeout is longer than the wait for response with retries` could (and IMO should) verify that error is bad request.

Also `"timeout during first request"` could verify both isTimeoutError() and isTemporaryError() predicates.

Reply:
Done